src/test/java/uk/gov/hmcts/reform/managecase/client/datastore/ChangeOrganisationRequestTest.java

```diff
@@ -0,0 +1,89 @@
+package uk.gov.hmcts.reform.managecase.client.datastore;
```

Comment:
Move to the package that corresponds to the model under test: i.e. `uk.gov.hmcts.reform.managecase.domain`.